update quickstart+101 (#1466)
meilisearch/documentation: learn/getting_started/quick_start.md

```bash
# Add Meilisearch package
sudo echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list

# Update APT and install Meilisearch

> sudo apt update && apt install meilisearch-http

Reviewer: ```suggestion
sudo apt update && sudo apt install meilisearch-http
```

# Launch Meilisearch

> sudo meilisearch

Reviewer: I don't think a sudo is necessary here; could be wrong.

```suggestion
meilisearch
```

```bash

> git clone https://github.com/meilisearch/Meilisearch

Reviewer: GitHub links are case-insensitive. However the folder that is generated when you run `git clone` will be based on the last piece of text in the link, e.g. the above code generates a folder called `Meilisearch`.

As @guimachiavelli mentioned recently, certain Unix-based OS can complain if you have a folder called `Meilisearch` and try to `cd meilisearch`, because they are case-sensitive. So we have to be careful about this. As a good rule of thumb, the URL should always be all-lowercase, and the folder you `cd` to as well.

```suggestion
git clone https://github.com/meilisearch/meilisearch
```

cd meilisearch
```

> In the cloned repository, choose the release you want to use. You can find the full list [here](https://github.com/meilisearch/MeiliSearch/releases).

Reviewer: ```suggestion
Choose the release you want to use. You can find the full list [here](https://github.com/meilisearch/meilisearch/releases).
```

> Now, run the following command with the tag you selected:

Reviewer: ```suggestion
In the cloned repository, run the following command replacing `vX.Y.Z` with the tag you selected:
```

> ```bash
> git checkout v0.25.2
> 
> #Compile Meilisearch using:

Reviewer: I don't think this was intentional? 👀 

````suggestion
```bash
git checkout v0.25.2
```

Finally, update the rust toolchain, compile the project, and execute the binary.
````

The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a developer tool that generates all the components needed to start searching.

### Let's try it!

1. Create an `html` file, for example, `index.html`
2. Open it in a text editor (e.g. Notepad, Sublime Text, Visual Studio Code)
3. Copy-paste any of the code examples above and save the file
4. Open `index.html` in your browser (double click on it in your folder)

> You should now have a working front-end search interface 🚀🔥

Reviewer: We should delete these lines as well. We can replace them with a line like this:

```
To learn more about instant-meilisearch, refer to its [documentation](https://github.com/meilisearch/instant-meilisearch#readme) located in the project repository.
```
